src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -59,6 +55,36 @@ public TimeSpan AutomaticRefreshInterval
         /// </summary>
         public static readonly TimeSpan DefaultRefreshInterval = new TimeSpan(0, 0, 5, 0);
 
+        /// <summary>
+        /// The default constructor.
+        /// </summary>
+        public BaseConfigurationManager()
+        {
+            _lastKnownGoodConfigurationCache = new EventBasedLRUCache<BaseConfiguration, DateTime>(
+                LKGConfigurationCacheOptions.DefaultLastKnownGoodConfigurationSizeLimit,
+                TaskCreationOptions.None,
+                new BaseConfigurationComparer(),
+                true);
+        }
+
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="options">The event queue task creation option.</param>
+        /// <param name="removeExpiredValues">Whether or not to remove expired items.</param>
+        /// <param name="taskCreationOptions">The event queue task creation option, default to None instead of LongRunning as LongRunning will always start a task on a new thread instead of ThreadPool.</param>
+        public BaseConfigurationManager(LKGConfigurationCacheOptions options, bool removeExpiredValues = true, TaskCreationOptions taskCreationOptions = TaskCreationOptions.None)
```

Comment:
would it make sense to add two properties to LKGConfigurationCacheOptions.
bool RemoveExpiredValues {default true}
TaskCreationOptions TaskCreationOptions {default TaskCreationOptions.None}

Reply:
God suggestion, thanks.